File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    isP = isY = 0
    for ele in s:
        if ele == 'P' or ele == 'p':
            isP += 1
        if ele == 'Y' or ele == 'y':
            isY += 1
    return True if isP == isY else False

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
  let [cntP, cntY] = [0, 0];

  [...s].forEach(ele => {
    if(ele === 'p' || ele === 'P') cntP++
    if(ele === 'y' || ele === 'Y') cntY++
  })
  return cntP === cntY ? true : false;
}